src/repositories/user.repository.ts

```diff
@@ -47,7 +47,8 @@ export const updateOne = async (
       password: data.password && hashPassword(data.password),
       profile: (data.profile || file) && {
         ...data.profile,
-        avatar: file && file?.buffer.toString("base64"),
+        avatar:
+          file && `data:image/png;base64,${file?.buffer.toString("base64")}`,
```

Comment:
Why did you do that? Why "data:image/png;base64"? why "png"?

Reply:
@VitaliiDenysyuk can we somehow get the format of the incoming avatar to apply to this line?

Reply:
@DavidKorystov,  I did not analyze all logic but it is very strange when you take something from database and add some string. And how do you know what type data in it?

Reply:
`file` should contain information about mime type

Reply:
something like `data:image/${file.mimetype};base64,${file?.buffer.toString("base64")}`